tap_github_search/tests/test_search_count_streams.py

```diff
@@ -2,9 +2,11 @@
 
 from datetime import date
 import os
+import types
 import logging
 from unittest.mock import Mock, patch
 import pytest
+import requests
```

Comment:
Why were these imports added? Was `try-except` hiding problems before?

Reply:
No, I hadn't tested after the last round of changes I made. Will work on adding the tests to CI.

One test was incorrectly skipped. Fixed that too:

```
mahangu@galactica tap-github % test_venv/bin/pytest -q tap_github_search/tests
/Users/mahangu/a8c/tap-github/test_venv/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
........................                                                                                                                                         [100%]
24 passed in 0.04s
```